test/integration/patients/sidebar/action-sidebar.js

```diff
@@ -1768,7 +1768,254 @@ context('action sidebar', function() {
+      .find('button')
+      .should('have.length', 5);
+
+    cy
+      .get('[data-attachments-files-region]')
+      .find('.js-remove');
+
+    cy
+      .routeAction(fx => {
+        fx.data = {
+          id: '2',
+          attributes: {
+            name: 'Owned by another team',
+          },
+          relationships: {
+            owner: { data: { id: '22222', type: 'teams' } },
+            state: { data: { id: '33333' } },
+            files: { data: [{ id: '1' }] },
+          },
+        };
+
+        return fx;
+      });
+
+    cy
+      .get('.patient__list')
+      .find('.table-list__item')
+      .as('listItems')
+      .eq(1)
+      .find('.patient__action-name')
+      .click()
+      .wait('@routeAction');
+
+    cy
+      .get('[data-action-region]')
+      .find('.js-input')
+      .should('have.length', 0);
+
+    cy
+      .get('[data-action-region]')
+      .find('button')
+      .should('have.length', 0);
+
+    cy
+      .get('[data-action-region]')
+      .should('contain', 'Permissions')
+      .and('contain', 'You are not able to change settings on actions.');
+
+    cy
+      .get('[data-attachments-files-region]')
+      .find('.js-remove')
+      .should('not.exist');
+
+    cy
+      .routeAction(fx => {
+        fx.data = {
+          id: '3',
+          attributes: {
+            name: 'Owned by team member',
+          },
+          relationships: {
+            owner: { data: { id: '22222', type: 'clinicians' } },
+            state: { data: { id: '33333' } },
+            files: { data: [{ id: '1' }] },
+          },
+        };
+
+        return fx;
+      });
+
+    cy
+      .get('@listItems')
+      .eq(2)
+      .find('.patient__action-name')
+      .click()
+      .wait('@routeAction');
+
+    cy
+      .get('[data-action-region]')
+      .find('.js-input')
+      .should('have.length', 2);
+
+    cy
+      .get('[data-action-region]')
+      .find('button')
+      .should('have.length', 5);
+
+    cy
+      .get('[data-attachments-files-region]')
+      .find('.js-remove');
+
+    cy
+      .routeAction(fx => {
+        fx.data = {
+          id: '4',
+          attributes: {
+            name: 'Owned by non team member',
+          },
+          relationships: {
+            owner: { data: { id: '33333', type: 'clinicians' } },
+            state: { data: { id: '33333' } },
+            files: { data: [{ id: '1' }] },
+          },
+        };
+
+        return fx;
+      });
+
+    cy
+      .get('@listItems')
+      .last()
+      .find('.patient__action-name')
+      .click()
+      .wait('@routeAction');
+
+    cy
+      .get('[data-action-region]')
+      .find('.js-input')
+      .should('have.length', 0);
+
+    cy
+      .get('[data-action-region]')
+      .find('button')
+      .should('have.length', 0);
+
+    cy
+      .get('[data-action-region]')
+      .should('contain', 'Permissions')
+      .and('contain', 'You are not able to change settings on actions.');
```

Comment:
Do we want to get more specific with this permission message?

In this `work:team:manage` scenario, we'll be showing a `'You are not able to change settings on actions.'` message to the user. 

Which is misleading because the user can edit other actions, just not this one specifically. This message implies they can't edit any actions.

Not a change for this PR/story, but something I noticed along the way.

![Screenshot 2023-09-06 at 3 24 22 PM](https://github.com/RoundingWell/care-ops-frontend/assets/35355575/3be815a8-a30b-4683-8a17-04ed12134c13)

Reply:
Interestingly that was true when we added the message the first time as it was with when we added `work:owned:manage` permission. Should like be "on this action" instead of "on actions"

Reply:
Agree that the messaging should be specific to _this_ action, instead of a general statement about actions.